File: Web_Client/src/app/models/Recruitment.ts
export interface Recruitment {
    id: number;
    address: string;
    createdAt: Date;
    description: string;
    experience: string;
    quantity: number;
    rank: string;
    salary: number;
    status: number;
    title: string;
    type: string;
    view: number;
    deadline: string;
    categoryId?: number;
    companyId?: number;
    companyName: string;
    categoryName: string;
}

File: Web_Sever/Web_Server/Models/Recruitment.cs
namespace Web_Server.Models
{
    public class Recruitment
    {
        public int Id { get; set; }
        public string Address {  get; set; }
        public DateTime CreatedAt { get; set; }
        public string Description { get; set; }
        public string Experience { get; set; }
        public int Quantity { get; set; }
         public string Rank { get; set; }
        public double Salary { get; set; }

        public int Status {  get; set; }

        public string Title { get; set; }
        public string Type {  get; set; }
        public int View {  get; set; }
        public string Deadline { get; set; }
        public Company Company { get; set; }
        public int CompanyId { get; set; }
        public Category Category { get; set; }
        public int CategoryId { get; set; }


    }
}
